examples/ArduinoIoTCloud-Advanced/arduino_secrets.h

```diff
@@ -1,6 +1,3 @@
-#include <ArduinoIoTCloud.h>
-#include <Arduino_ConnectionHandler.h>
```

Comment:
Although it is not technically required, I think the code is more intuitive if the `#include` directive for `Arduino_ConnectionHandler.h` is left in place in the `arduino_secrets.h` files even after it is also added to `thingProperties.h`.

The reason is that this makes it easier for the user to understand the origin of the `BOARD_HAS_*` macros referenced in the file.

Reply:
should be fixed now! Thanks @per1234